<launch>
	<!-- run the position_control.cpp -->

	<node pkg="drone_command" type="px4_pos_estimator" name="px4_pos_estimator" output="screen">

		<!-- load blacklist, config -->
                <rosparam command="load" file="$(find drone_command)/config/Parameter_for_control.yaml" />
	</node>
	<node pkg="drone_command" type="gazebo_ground_truth" name="gazebo_ground_truth" output="screen">

	
	</node>
</launch>
